From 4431508e64d25d8d5a18b0c369f33d48e83a6f82 Mon Sep 17 00:00:00 2001
From: Naresh Solanki <naresh.solanki@9elements.com>
Date: Tue, 31 Oct 2023 18:24:54 +0000
Subject: [PATCH 09/13] configuration: sbp1: Configure PowerState of some
 suplies

Some sensor show up as critical when host is On.
Configure below supplies PowerState as On.
P1V8_PCH_AUX
PVCORE_NIC2
PVCORE_NIC1
P3V3_NIC
P1V2_NIC
P1V8_NIC
P1V05_PCH_AUX
PVCCFA_EHV_CPU0
PVCCINFAON_CPU0
PVCCFA_EHV_CPU1
PVCCINFAON_CPU1
PVCCFA_EHV_CPU2
PVCCINFAON_CPU2
PVCCFA_EHV_CPU3
PVCCINFAON_CPU3
PVCCIN_CPU0
PVCCFA_EHV_FIVRA_CPU0
PVCCIN_CPU1
PVCCFA_EHV_FIVRA_CPU1
PVCCIN_CPU2
PVCCFA_EHV_FIVRA_CPU2
PVCCIN_CPU3
PVCCFA_EHV_FIVRA_CPU3
PVNN_MAIN_CPU0
PVNN_MAIN_CPU3
PVNN_MAIN_CPU1
PVNN_MAIN_CPU2

Signed-off-by: Naresh Solanki <naresh.solanki@9elements.com>
Change-Id: Ie7d50b3e9c121b8db6aeee0c7fbbdc0b71c6fd7d
---
 configurations/sbp1_baseboard.json | 120 ++++++++++++++---------------
 1 file changed, 60 insertions(+), 60 deletions(-)

diff --git a/configurations/sbp1_baseboard.json b/configurations/sbp1_baseboard.json
index 3e85131..b2ef3bc 100644
--- a/configurations/sbp1_baseboard.json
+++ b/configurations/sbp1_baseboard.json
@@ -382,7 +382,7 @@
             ],
             "Name": "P1V8_PCH_AUX",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -559,7 +559,7 @@
             ],
             "Name": "PVCORE_NIC2",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -625,7 +625,7 @@
             ],
             "Name": "PVCORE_NIC1",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -691,7 +691,7 @@
             ],
             "Name": "P3V3_NIC",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -757,7 +757,7 @@
             ],
             "Name": "P1V2_NIC",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -823,7 +823,7 @@
             ],
             "Name": "P1V8_NIC",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -889,7 +889,7 @@
             ],
             "Name": "P1V05_PCH_AUX",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -1243,7 +1243,7 @@
             ],
             "Name": "SSB_AUX",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_AUX PVDRV Output Current",
             "curr2_Name": "SSB_AUX P3V3_PCH Output Current",
@@ -1261,7 +1261,7 @@
             ],
             "Name": "SSB_RSSD03",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD03 P12V Output Current",
             "curr2_Name": "SSB_RSSD03 P3V3 Output Current",
@@ -1279,7 +1279,7 @@
             ],
             "Name": "SSB_RSSD02",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD02 P12V Output Current",
             "curr2_Name": "SSB_RSSD02 P3V3 Output Current",
@@ -1297,7 +1297,7 @@
             ],
             "Name": "SSB_RSSD01",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD01 P12V Output Current",
             "curr2_Name": "SSB_RSSD01 P3V3 Output Current",
@@ -1315,7 +1315,7 @@
             ],
             "Name": "SSB_RSSD04",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD04 P12V Output Current",
             "curr2_Name": "SSB_RSSD03 P3V3 Output Current",
@@ -1333,7 +1333,7 @@
             ],
             "Name": "SSB_RSSD05",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD05 P12V Output Current",
             "curr2_Name": "SSB_RSSD05 P3V3 Output Current",
@@ -1351,7 +1351,7 @@
             ],
             "Name": "SSB_RSSD08",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD08 P12V Output Current",
             "curr2_Name": "SSB_RSSD08 P3V3 Output Current",
@@ -1369,7 +1369,7 @@
             ],
             "Name": "SSB_RSSD07",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD07 P12V Output Current",
             "curr2_Name": "SSB_RSSD07 P3V3 Output Current",
@@ -1387,7 +1387,7 @@
             ],
             "Name": "SSB_RSSD06",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD06 P12V Output Current",
             "curr2_Name": "SSB_RSSD06 P3V3 Output Current",
@@ -1405,7 +1405,7 @@
             ],
             "Name": "SSB_RSSD14",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD14 P12V Output Current",
             "curr2_Name": "SSB_RSSD14 P3V3 Output Current",
@@ -1423,7 +1423,7 @@
             ],
             "Name": "SSB_RSSD13",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD13 P12V Output Current",
             "curr2_Name": "SSB_RSSD13 P3V3 Output Current",
@@ -1441,7 +1441,7 @@
             ],
             "Name": "SSB_RSSD12",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD12 P12V Output Current",
             "curr2_Name": "SSB_RSSD12 P3V3 Output Current",
@@ -1459,7 +1459,7 @@
             ],
             "Name": "SSB_RSSD11",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD11 P12V Output Current",
             "curr2_Name": "SSB_RSSD11 P3V3 Output Current",
@@ -1477,7 +1477,7 @@
             ],
             "Name": "SSB_RSSD10",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD10 P12V Output Current",
             "curr2_Name": "SSB_RSSD10 P3V3 Output Current",
@@ -1495,7 +1495,7 @@
             ],
             "Name": "SSB_RSSD09",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD09 P12V Output Current",
             "curr2_Name": "SSB_RSSD09 P3V3 Output Current",
@@ -1513,7 +1513,7 @@
             ],
             "Name": "SSB_RSSD15",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD15 P12V Output Current",
             "curr2_Name": "SSB_RSSD15 P3V3 Output Current",
@@ -1531,7 +1531,7 @@
             ],
             "Name": "SSB_RSSD16",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD16 P12V Output Current",
             "curr2_Name": "SSB_RSSD16 P3V3 Output Current",
@@ -1741,7 +1741,7 @@
             ],
             "Name": "SSB_RSSD19",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD19 P12V Output Current",
             "curr2_Name": "SSB_RSSD19 P3V3 Output Current",
@@ -1759,7 +1759,7 @@
             ],
             "Name": "SSB_RSSD18",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD18 P12V Output Current",
             "curr2_Name": "SSB_RSSD18 P3V3 Output Current",
@@ -1777,7 +1777,7 @@
             ],
             "Name": "SSB_RSSD17",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD17 P12V Output Current",
             "curr2_Name": "SSB_RSSD17 P3V3 Output Current",
@@ -1795,7 +1795,7 @@
             ],
             "Name": "SSB_RSSD20",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD20 P12V Output Current",
             "curr2_Name": "SSB_RSSD20 P3V3 Output Current",
@@ -1813,7 +1813,7 @@
             ],
             "Name": "SSB_RSSD21",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD21 P12V Output Current",
             "curr2_Name": "SSB_RSSD21 P3V3 Output Current",
@@ -1831,7 +1831,7 @@
             ],
             "Name": "SSB_RSSD22",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD22 P12V Output Current",
             "curr2_Name": "SSB_RSSD22 P3V3 Output Current",
@@ -1849,7 +1849,7 @@
             ],
             "Name": "SSB_RSSD24",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD24 P12V Output Current",
             "curr2_Name": "SSB_RSSD24 P3V3 Output Current",
@@ -1867,7 +1867,7 @@
             ],
             "Name": "SSB_RSSD23",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD23 P12V Output Current",
             "curr2_Name": "SSB_RSSD23 P3V3 Output Current",
@@ -1885,7 +1885,7 @@
             ],
             "Name": "SSB_RSSD25",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD25 P12V Output Current",
             "curr2_Name": "SSB_RSSD25 P3V3 Output Current",
@@ -1903,7 +1903,7 @@
             ],
             "Name": "SSB_RSSD26",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD26 P12V Output Current",
             "curr2_Name": "SSB_RSSD26 P3V3 Output Current",
@@ -1921,7 +1921,7 @@
             ],
             "Name": "SSB_RSSD27",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD27 P12V Output Current",
             "curr2_Name": "SSB_RSSD27 P3V3 Output Current",
@@ -1939,7 +1939,7 @@
             ],
             "Name": "SSB_RSSD32",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD32 P12V Output Current",
             "curr2_Name": "SSB_RSSD32 P3V3 Output Current",
@@ -1957,7 +1957,7 @@
             ],
             "Name": "SSB_RSSD31",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD31 P12V Output Current",
             "curr2_Name": "SSB_RSSD31 P3V3 Output Current",
@@ -1975,7 +1975,7 @@
             ],
             "Name": "SSB_RSSD30",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD30 P12V Output Current",
             "curr2_Name": "SSB_RSSD30 P3V3 Output Current",
@@ -1993,7 +1993,7 @@
             ],
             "Name": "SSB_RSSD29",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD29 P12V Output Current",
             "curr2_Name": "SSB_RSSD29 P3V3 Output Current",
@@ -2011,7 +2011,7 @@
             ],
             "Name": "SSB_RSSD28",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Type": "MAX5970",
             "curr1_Name": "SSB_RSSD28 P12V Output Current",
             "curr2_Name": "SSB_RSSD28 P3V3 Output Current",
@@ -2300,7 +2300,7 @@
             "Name": "PVCCFA_EHV_CPU0",
             "Name1": "PVCCINFAON_CPU0",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -2393,7 +2393,7 @@
             "Name": "PVCCFA_EHV_CPU1",
             "Name1": "PVCCINFAON_CPU1",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -2486,7 +2486,7 @@
             "Name": "PVCCFA_EHV_CPU2",
             "Name1": "PVCCINFAON_CPU2",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -2579,7 +2579,7 @@
             "Name": "PVCCFA_EHV_CPU3",
             "Name1": "PVCCINFAON_CPU3",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -2672,7 +2672,7 @@
             "Name": "PVCCIN_CPU0",
             "Name1": "PVCCFA_EHV_FIVRA_CPU0",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -2765,7 +2765,7 @@
             "Name": "PVCCIN_CPU1",
             "Name1": "PVCCFA_EHV_FIVRA_CPU1",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -2858,7 +2858,7 @@
             "Name": "PVCCIN_CPU2",
             "Name1": "PVCCFA_EHV_FIVRA_CPU2",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -2951,7 +2951,7 @@
             "Name": "PVCCIN_CPU3",
             "Name1": "PVCCFA_EHV_FIVRA_CPU3",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -3040,7 +3040,7 @@
             ],
             "Name": "PVNN_MAIN_CPU0",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -3108,7 +3108,7 @@
             ],
             "Name": "PVNN_MAIN_CPU3",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -3176,7 +3176,7 @@
             ],
             "Name": "PVNN_MAIN_CPU1",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -3244,7 +3244,7 @@
             ],
             "Name": "PVNN_MAIN_CPU2",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -3312,7 +3312,7 @@
             ],
             "Name": "PVPP_HBM_CPU3",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -3366,7 +3366,7 @@
             ],
             "Name": "PVPP_HBM_CPU1",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -3420,7 +3420,7 @@
             ],
             "Name": "PVPP_HBM_CPU2",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -3474,7 +3474,7 @@
             ],
             "Name": "PVPP_HBM_CPU0",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -3528,7 +3528,7 @@
             ],
             "Name": "PVCCD_HV_CPU3",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -3591,7 +3591,7 @@
             ],
             "Name": "PVCCD_HV_CPU1",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -3654,7 +3654,7 @@
             ],
             "Name": "PVCCD_HV_CPU2",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -3717,7 +3717,7 @@
             ],
             "Name": "PVCCD_HV_CPU0",
             "PollRate": 5.0,
-            "PowerState": "Always",
+            "PowerState": "On",
             "Thresholds": [
                 {
                     "Direction": "greater than",
-- 
2.41.0

